scripts/update-gerbil-nix-recipe.ss

```diff
@@ -274,7 +307,8 @@
       checkouts-dir: checkouts-dir
       source-dir: smug-gerbil-dir
       nixpkgs-dir: nixpkgs-dir
-      stable: stable))
+      stable: stable
+      cmdopts: ["--smug-gerbil-dir" "--gerbil-off" "--stable"]))
```

Comment:
While we're at it, shouldn't we uniformly add the -repo and -dir arguments everywhere?

Reply:
Sure I can do that.